_pkgdown.yml

```
@@ -55,6 +54,7 @@ reference:
       - col_relabel
       - example_cdisc_data
   - title: Deprecated functions and methods
-    desc: Functions deprecated in teal.data v0.3.1
+    desc: Functions deprecated in teal.data
```

Where else would they be deprecated?
```suggestion
    desc: Deprecated functions.
```

---

```
- title: Deprecated functions and methods
    desc: Deprecated functions.
```

"This looks odd, perhaps?"
```
- title: Deprecated
    desc: Deprecated  functions and methods
```

---

:+1:

---

Oh, actually, is `desc` required? If not, maybe just drop it?

---

its not.